core/governance/market.go

```diff
@@ -373,18 +373,22 @@ func validateAssetBasic(assetID string, assets Assets, deepCheck bool) (types.Pr
 		return types.ProposalErrorUnspecified, nil
 	}
 
-	_, err := assets.Get(assetID)
+	as, err := assets.Get(assetID)
 	if err != nil {
 		return types.ProposalErrorInvalidAsset, err
 	}
 	if !assets.IsEnabled(assetID) {
 		return types.ProposalErrorInvalidAsset,
 			fmt.Errorf("asset is not enabled %v", assetID)
 	}
+	if positionDecimals > int64(as.DecimalPlaces()) {
```

Comment:
Is the code doing the validation here just for spot market for futures / perps too?

Reply:
Looks like `validateAssetBasic` is only called for spot markets

Reply:
it's only for baseAsset, i.e. spots.

Reply:
Cheers!